### PR TITLE
Added a better, prettier error page (Fixes #120)

### LINKED ISSUE #120: Better, Prettier Error Pages
Flask supports really easy error handling that we should definitely implement.

```
@app.errorhandler(404)
def page_not_found(e):
    return render_template('404.html'), 404
```

### PR DESCRIPTION
**Pull request by RamZallan**
Fixes #120

### CODE REVIEW

**Comment by mbillow** [comment]
What does this look like on a 500?

**Comment by RamZallan** [comment]
@mbillow:
[Looks like](http://flask.pocoo.org/docs/0.12/patterns/errorpages/) that on a 500 error, Flask just passes in "500: Internal Server Error" in place of "404: Not Found", so I think it would look something like this:
image

File: conditional/__init__.py [hunk 1]
```diff
@@ -63,6 +64,20 @@ def static_proxy(path):
...
+    member = ldap_get_member(username)
+    data = dict()
+    data['username'] = member.uid
+    data['name'] = member.cn
+    code = error.code
+    return render_template(request=request,
+                            template_name='404.html',
+                            error=str(error),
+                            error_code=code,
+                            **data), 404
```
Comment (audiolion) [inline conditional/__init__.py]: reference to error code in return is http://flask.pocoo.org/docs/0.12/api/#flask.Flask.errorhandler

also a small thing but if the template handles 404 and 500 should it be called `404.html`? maybe `error.html`

Comment (RamZallan) [inline conditional/__init__.py]: Will fix both, good points!

**Pull request merged**